recipes/ptac/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ptac" %}
+{% set version = "0.1.2a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptac-{{ version }}.tar.gz
+  sha256: 8bbf3ef7f295d06266e1a2438dd2841ab61c02bc82169311fac8e5bba1496b8d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools ==63.4.1
+    - wheel ==0.37.1
+
+  host:
+    - python >=3.8
+    - pip
```

Comment:
If these are needed, they should all be under `host` and eliminate the `build` section. Also, does it really need specific versions of setuptools and wheel to build the package?

Reply:
@dopplershift Thanks for the feedback! I just updated the requirements